src/styles/_utility.scss

```diff
@@ -353,6 +354,57 @@ $skeleton-height-map: (
   ),
 );
 
+// for now, we're only doing fixed witdth for tablet & desktop.
+// because mobile is relatively more stable compared to desktop
+// TODO: implement fixed on mobile as well
+$line-width-map: (
+  tajweed: (
+    tablet: (
+      1: calc(12.5 * var(--spacing-mega)),
+      2: calc(15.5 * var(--spacing-mega)),
+      3: calc(18 * var(--spacing-mega)),
+      4: calc(20.5 * var(--spacing-mega)),
+      5: calc(22 * var(--spacing-mega)),
+    ),
+  ),
+  qpc_uthmani_hafs: (
+    tablet: (
+      1: 55vh,
+      2: 59vh,
+      3: 63.8vh,
+      4: 65vh,
+      5: 66vh,
+    ),
+  ),
+  text_indopak: (
```

Comment:
I think we should separate this into 15 and 16 because the scales will not be 100% identical